packages/store-ui/src/ProductQuantity/useNumericStepper.ts

```diff
@@ -19,6 +19,10 @@ export const useNumericStepper = ({
 }: Options) => {
   const [value, setValue] = useState(() => narrow(initialValue, min, max))
 
+  useEffect(() => {
+    setValue(narrow(initialValue, min, max))
```

Comment:
What change is that?

Reply:
That initialValue change, the numeric stepper value should also change